tracing-subscriber/src/fmt/format/pretty.rs

```diff
@@ -0,0 +1,342 @@
+use super::*;
+use crate::{
+    field::{VisitFmt, VisitOutput},
+    fmt::fmt_subscriber::{FmtContext, FormattedFields},
+    registry::LookupSpan,
+};
+
+use std::{
+    fmt::{self, Write},
+    iter,
+};
+use tracing_core::{
+    field::{self, Field},
+    Collect, Event, Level,
+};
+
+#[cfg(feature = "tracing-log")]
+use tracing_log::NormalizeEvent;
+
+use ansi_term::{Colour, Style};
+
+/// An excessively pretty, human-readable event formatter.
```

Comment:
I personally _love_ this futch formatter, but I'm not sure whether "excessively" is the right word.

Reply:
i was trying to be cute. we can remove it. imo the focus on aesthetics over everything else is a little excessive for most use cases though

Reply:
that's fair, let's keep it!